testing/functional_tests/test_segment_volume.py

```diff
@@ -133,6 +133,51 @@ def test_segment_volume_2d(download_functional_test_files):
     shutil.rmtree(PATH_MODEL)
 
 
+def test_segment_volume_2d_no_prepro_transform(download_functional_test_files):
+    model = imed_models.Unet(in_channel=1,
+                             out_channel=1,
+                             depth=2,
+                             drop_rate=DROPOUT,
+                             bn_momentum=BN)
+
+    if not os.path.exists(PATH_MODEL):
+        os.mkdir(PATH_MODEL)
```

Comment:
I would recommend to use either [`os.makedirs`](https://docs.python.org/3/library/os.html#os.makedirs) or even better pathlib's [`mkdir`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir) (with parents=True) such that if the parent path doesn't exist, they will be created automatically.

Reply:
Good to know! This is a bit out of scope for this PR so I suggest we make the correction to this test and all the other tests from this file in a separate PR.